tests/Gedmo/Tree/MaterializedPathORMTest.php

```diff
@@ -165,4 +165,13 @@ protected function getUsedEntityFixtures(): array
             self::CATEGORY,
         ];
     }
+
+    private function generateTreeNodeValue(MPCategory $category): string
```

Comment:
IMO, this method doesn't seem to generate a value, but instead performs a find or get operation.